www/templates/survey/enketo/add-note-button.html

```diff
@@ -2,5 +2,5 @@
   <!-- <button class="button ion-plus diary-btn" ng-click="openPopover($event, timelineEntry, 'NOTES')">
     {{displayLabel}}
   </button> -->
-  <diary-button text="displayLabel"></diary-button>
-</div> 
\ No newline at end of file
+  <diary-button text="displayLabel" ng-click="openPopover($event, timelineEntry, 'NOTES')"></diary-button>
+</div> 
```

Comment:
Note to self - the `ng-click` was removed in
https://github.com/JGreenlee/e-mission-phone/commit/f4346d0e5b154bae3b081258156a9971bff931de#diff-7699c6da2de00352d238d5a4eb12ff678300c7ab006a60dfe94e1068c1417da5

while switching to react.

Added back now that enketo is fixed